### PR TITLE
feat: Adds automatic jaas account creation.

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

File: resources/register-jaas-account.sh
Comment (saghul) [inline resources/register-jaas-account.sh:14]: I think the usual place to put this is in .well-known/

File: resources/register-jaas-account.sh
Comment (saghul) [inline resources/register-jaas-account.sh:21]: Can the error be read from the response payload?

**Review by saghul — COMMENTED** [review]
A couple of drive-by comments

File: resources/register-jaas-account.sh
Comment (damencho) [reply resources/register-jaas-account.sh:14]: This is where the service is looking for it?
@oanaianc @raluca8x8 ?

File: resources/register-jaas-account.sh
Comment (damencho) [reply resources/register-jaas-account.sh:21]: Yep, I will add and that to the print.

File: resources/register-jaas-account.sh
Comment (damencho) [reply resources/register-jaas-account.sh:14]: .well-known is handled only for port 80 (https://github.com/jitsi/jitsi-meet/blob/779ecd6da640c9f2ff0393355f5cd97db290776b/doc/debian/jitsi-meet/jitsi-meet.example#L23), where the service requires the https to be in place ... but I guess with .txt it will work and under https

Comment (damencho) [reply resources/register-jaas-account.sh:14]: But cannot change it till the service is updated.

Comment (oanaianc) [reply resources/register-jaas-account.sh:14]: > This is where the service is looking for it? @oanaianc @raluca8x8 ?

yes, we check for the challenge in jitsi-chalenge.txt

Comment (damencho) [reply resources/register-jaas-account.sh]: @saghul I got confirmation from @oanaianc that she will do the change to the service. So you can approve if you agree with the changes and I will merge it once I have the green light from @oanaianc .

**Review by oanaianc — APPROVED** [review]

**Pull request merged**
